scripts/upgrade

```diff
@@ -118,16 +134,34 @@ ynh_script_progression --message="Installating composer and dependencies..."
 
 ynh_install_composer --phpversion="$phpversion" --workdir="$final_path"
 
+if [[ -f "$final_path/config/config.ini.php" ]]; then
+	pushd "$final_path"
+		ynh_exec_as $app ./console core:update
+	popd
+fi
+
+#=================================================
+# INSTALL MISSING ICONS
+#=================================================
+
+if [ ! "$(ls -A "$final_path/plugins/Morpheus/icons")" ]
+then
+    git clone https://github.com/matomo-org/matomo-icons.git "$final_path/plugins/Morpheus/icons" --quiet
```

Comment:
why not using something like `ynh_setup_source --dest_dir="$final_path/plugins/Morpheus/icons" --source_id="icons"` adn having an icons.src file ?

Reply:
Thanks @yalh76 for reviewing this PR. I added the download from `ynh_setup_source`